examples/pbsam-barn_bars/barn_bars_electro.in

```diff
@@ -26,11 +26,15 @@ elec name comp_solv        # Gly
     temp 298.15                                       # System temperature (K)
     pdie 4.0                                          # Solute dielectric                 
     sdie 78.0                                         # Solvent dielectric                
-    ion charge 0.0 conc 0.10 radius 0.0               # Monovalent salt conc in M
-
-    dime 150
-    write pot dx barn_bars_0.1M.dx
-    write 3dmap  barn_bars_0.1M.map
+    # WARN: ion is not supported in pbam-auto
+    # ion charge 0.0 conc 0.10 radius 0.0               # Monovalent salt conc in M
+
+    # WARN: dime is not supported in pbam-auto
+    # dime 150
```

Comment:
@sobolevnrm should `dime` be part of the pbsam-auto specification? It is also not documented.

Reply:
I don't think `dime` has a role in pbsam.